daemon/policy_test.go

```diff
@@ -36,71 +36,16 @@ var (
 	IPv4Addr, _ = addressing.NewCiliumIPv4("10.11.12.13")
 )
 
-func (ds *DaemonSuite) TestFindNode(c *C) {
-	var nullPtr *policy.Node
-
-	pn := policy.Node{
-		Name: "io.cilium",
-		Children: map[string]*policy.Node{
-			"foo": {},
-			"bar": {},
-		},
-	}
-
-	err := ds.d.PolicyAdd("io.cilium", &pn)
-	c.Assert(err, Equals, nilApiError)
-
-	n, p := ds.d.findNode("io.cilium")
-	c.Assert(n, Not(Equals), nil)
-	c.Assert(p, Equals, nullPtr)
-
-	n, p = ds.d.findNode("io.cilium.foo")
-	c.Assert(n, Not(Equals), nil)
-	c.Assert(p, Not(Equals), nil)
-
-	n, p = ds.d.findNode("io.cilium.baz")
-	c.Assert(n, Equals, nullPtr)
-	c.Assert(p, Equals, nullPtr)
-
-	n, p = ds.d.findNode("io.cilium..foo")
-	c.Assert(n, Not(Equals), nullPtr)
-	c.Assert(p, Not(Equals), nullPtr)
-
-	err = ds.d.PolicyDelete("io.cilium", "")
-	c.Assert(err, IsNil)
-}
-
-func (ds *DaemonSuite) TestPolicyGet(c *C) {
-	pn := policy.Node{
-		Name: "io.cilium",
-		Children: map[string]*policy.Node{
-			"foo": {
-				Name: "magic",
-			},
-		},
-	}
-
-	err := ds.d.PolicyAdd("io.cilium", &pn)
-	c.Assert(err, Equals, nilApiError)
-
-	n, p := ds.d.findNode("io.cilium.foo")
-	c.Assert(n, Not(Equals), nil)
-	c.Assert(p, Not(Equals), nil)
-
-	err = ds.d.PolicyDelete("io.cilium.foo", "")
-	c.Assert(err, IsNil)
-}
-
 func (ds *DaemonSuite) TestUpdateConsumerMap(c *C) {
-	lblProd := labels.NewLabel("io.cilium.Prod", "", common.CiliumLabelSource)
-	lblQA := labels.NewLabel("io.cilium.QA", "", common.CiliumLabelSource)
-	lblFoo := labels.NewLabel("io.cilium.foo", "", common.CiliumLabelSource)
-	lblBar := labels.NewLabel("io.cilium.bar", "", common.CiliumLabelSource)
-	lblJoe := labels.NewLabel("io.cilium.user", "joe", common.CiliumLabelSource)
-	lblPete := labels.NewLabel("io.cilium.user", "pete", common.CiliumLabelSource)
+	lblProd := labels.NewLabel("root.Prod", "", common.CiliumLabelSource)
```

Comment:
Ah, so the user type `docker --label "Prod"` and the label will actually be `root.Prod`?